projects/base-library/README.md

```diff
@@ -1,15 +1,65 @@
+
+```board.h``` - provides basic functions for CAN, I2C, ADC, PWM, and UART communication
+
+```debug.h``` - provides debugging functions such as printing/storing logs
+
+```error.h``` - provides functions that properly handle errors in operations such as CAN communication or issues with receiving sensor data
+
+```config.h``` - specifies CAN/I2C frames, PWM rates, and defines what is being used, so that the rest can be excluded - NOTE: this file is to be edited by the user
+
+As you probably notice, the location of each feature is quite intuitive, especially once you've seen the diagram above.
+
+To find these files, navigate through the repository as follows: ```projects -> base-library -> project -> Core -> Inc -> xxxxx.h```
+
+## User Manual
+Before continuing, please read the "What is the Base Library" paragraph above if you haven't already.
+
+The three boxes at the top of the firmware design diagram illustrate the tasks of the individual teams. To reach the desired functionality for a specific COM module, such as reading data from a sensor, teams have to determine which functions are required, and what the proper inputs to those functions would be. Most important for this would be the ```board.h``` file, but the comprehensive list above covers everything available.
+
+There are also features that you can request from the firmware team. If there is any specific functionality you would like for handling errors, such as turning off a sensor if we can no longer receive data from it, then leave your request on the [Confluence feature request page](https://ubcsailbot.atlassian.net/wiki/spaces/prjt22/pages/1994457093/Feature+Request). Additionally, any other requests for code that needs new functions/functionality that isn't included in the base library can also be left in the same place. 
+
+Lastly, you can change values in the configurations file based on what you need and don't need from the base library. This is an important consideration as it will improve the performance of your COM Module!
+
+## Repository Structure
+
+```
+projects/
+    base-library/
+        project/
+            Core/
+                Inc                        - Contains base library header files
+                Src                        - Contains base library source files
+        tests
+        README.md                          - Base library user instructions (you're here right now)
+    rudder-controller
+    template                               - Template to be copied for teams writing their own code
+shared-docs
+tutorials/
+    STM32F407_demo/                        - Comprehensive demo project
+        README.md                          - Start here for the demo
+    setup.md                               - Tutorial for STM32CubeIDE setup
+    testing.md                             - Tutorial for how to create tests
+.gitignore
+LICENSE
+README.md                                  - Relevant background info on COM Modules
+```
```

Comment:
Don't need this here. It is already there in the root README. Only baselib specific info should be written here.